toolkit/scripts/toolchain/container/mount_chroot_start_build.sh

```diff
@@ -8,36 +8,43 @@ if [[ -z "$LFS" ]]; then
 fi
 echo LFS root is: $LFS
 
+# 7.2. Changing Ownership
 # Change temp tools to root ownership
-chown -R root:root $LFS/tools
+chown -R root:root $LFS/{usr,lib,var,etc,bin,sbin,tools}
+case $(uname -m) in
+  x86_64) chown -R root:root $LFS/lib64 ;;
+esac
 
+# 7.3. Preparing Virtual Kernel File Systems
 mkdir -pv $LFS/{dev,proc,sys,run}
 mknod -m 600 $LFS/dev/console c 5 1
 mknod -m 666 $LFS/dev/null c 1 3
 mount -v --bind /dev $LFS/dev
-mount -vt devpts devpts $LFS/dev/pts -o gid=5,mode=620
+#mount -vt devpts devpts $LFS/dev/pts -o gid=5,mode=620
```

Comment:
Assume this is ok

Reply:
I've filed a bug to track this: https://microsoft.visualstudio.com/OS/_workitems/edit/47485967